examples/src/main/scala/org/apache/spark/examples/mllib/HypothesisTestingExample.scala

```diff
@@ -68,7 +68,7 @@ object HypothesisTestingExample {
     // against the label.
     val featureTestResults: Array[ChiSqTestResult] = Statistics.chiSqTest(obs)
     featureTestResults.zipWithIndex.foreach { case (k, v) =>
-      println("Column " + (v + 1).toString + ":")
+      println(s"Column ${(v + 1).toString} :")
```

Comment:
.toString is redundant here and elsewhere with interpolation. I think that should be simplified.

Reply:
@srowen Thanks , Changes addressed